src/textbooks/Textbooks.jsx

```diff
@@ -0,0 +1,147 @@
+    title: processingNotificationTitle,
+  } = useSelector(getProcessingNotification);
+
+  if (isLoading) {
+    return (
+      <Row className="m-0 mt-4 justify-content-center">
+        <LoadingSpinner />
+      </Row>
+    );
+  }
+
+  return (
+    <>
+      <Helmet>
+        <title>
+          {`${courseDetails?.name} | ${intl.formatMessage(messages.headingTitle)}`}
+        </title>
+      </Helmet>
+      <Container size="xl" className="px-4">
+        <section className="mb-4 mt-5">
+          <SubHeader
+            title={intl.formatMessage(messages.headingTitle)}
+            breadcrumbs={(
+              <Breadcrumb ariaLabel={intl.formatMessage(messages.breadcrumbAriaLabel)} links={breadcrumbs} />
+            )}
+            headerActions={(
+              <Button
+                iconBefore={AddIcon}
+                onClick={openTextbookForm}
+                disabled={isTextbookFormOpen}
+              >
+                {intl.formatMessage(messages.newTextbookButton)}
+              </Button>
+            )}
+          />
+          <Layout
+            lg={[{ span: 9 }, { span: 3 }]}
+            md={[{ span: 9 }, { span: 3 }]}
+            sm={[{ span: 12 }, { span: 12 }]}
+            xs={[{ span: 12 }, { span: 12 }]}
+            xl={[{ span: 9 }, { span: 3 }]}
+          >
+            <Layout.Element>
+              <article>
+                <section className="textbook-section">
+                  <div className="pt-4">
+                    {textbooks.length ? textbooks.map((textbook, index) => (
+                      <TextbookCard
+                        key={textbook.id}
+                        textbook={textbook}
+                        courseId={courseId}
+                        handleSavingStatusDispatch={handleSavingStatusDispatch}
+                        onEditSubmit={handleTextbookEditFormSubmit}
+                        onDeleteSubmit={handleTextbookDeleteSubmit}
+                        textbookIndex={index}
+                      />
+                    )) : (
+                      !isTextbookFormOpen && <EmptyPlaceholder onCreateNewTextbook={openTextbookForm} />
+                    )}
+                    {isTextbookFormOpen && (
+                      <TextbookForm
+                        closeTextbookForm={closeTextbookForm}
+                        initialFormValues={getTextbookFormInitialValues()}
+                        onSubmit={handleTextbookFormSubmit}
+                        onSavingStatus={handleSavingStatusDispatch}
+                        courseId={courseId}
+                      />
+                    )}
+                  </div>
+                </section>
+              </article>
+            </Layout.Element>
+            <Layout.Element>
+              <TextbookSidebar courseId={courseId} />
+            </Layout.Element>
+          </Layout>
+        </section>
+      </Container>
+      <ProcessingNotification
+        isShow={isShowProcessingNotification}
```

Comment:
```suggestion
        isShow={showProcessingNotification}
```

(I have a similar issue with the `isShow` property in `<ProcessingNotification />`, but that's a refactor for another day.)

Reply:
Fixed